pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -68,7 +69,10 @@ type provider interface {
 }
 
 type connectionStore interface {
-	GetConnectionRecord(connectionID string) (*ConnectionRecord, error)
+	saveConnectionRecord(record *ConnectionRecord) error
```

Comment:
Why do we have un-exported methods in an interface, let's discuss

Reply:
removed the interface, i dont think it was required